luis-delete.js

```diff
@@ -0,0 +1,40 @@
+function del(){
+    var rp = require('request-promise');
+    var fse = require('fs-extra');
+    var path = require('path');
+    const LUIS_programmaticKey = "67fa05fd36ca4d4cbc8c2eb91e41dc10";
```

Comment:
You are repeatedly using these three values, so you can put it into a config file, then import it into these files. In that way, when you change some of the value, you don't need to change it in three places, just one place. This is a common best practice, recommend you to follow this way.